jc 1.13.1

kellyjonbrazil opened the pull request:
bump to v1.13.1

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Submitting manually since `brew bump-formula-pr --strict jc --url=https://files.pythonhosted.org/packages/ef/95/d1b5aa0662572629110fb73c3730c4664040fbc2adc92f4e2d04ec789642/jc-1.13.1.tar.gz --sha256=0d743199ed58c3ce09f4171c97e5aaa7df91fc42bc18854a773f2e19041d9d1a` is giving me the following error: 
```
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from f4f4d911c to c115d59b2.
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 1 formula.

==> replace /https:\/\/files\.pythonhosted\.org\/packages\/bf\/34\/1145fb85a333524b06d562f803a85e
==> replace "b7ffbc90720681fd09033e21301471b673cebfaa29016cdd01630055e3c9226d" with "0d743199ed58
==> brew update-python-resources jc
Error: Unable to determine dependencies for "jc" because of a failure when running
`pipgrip --json jc==1.13.1`. Please update the resources for "jc" manually.
```

dawidd6 commented:
cc @Rylan12

Rylan12 commented:
Thanks, @kellyjonbrazil.

There are some formulae that can't have their resources automatically updated, but `jc` shouldn't have this issue. It seems to work fine for me locally.

Can you post the output of: `brew info pipgrip` and `pipgrip --json jc==1.13.1 -vv`?

kellyjonbrazil commented:
Yeah, I've done this before, but haven't tried it in a while. Here's the output you requested:

```
kbrazil-mac:~ kbrazil$ brew info pipgrip
pipgrip: stable 0.5.1 (bottled)
Lightweight pip dependency resolver
https://github.com/ddelange/pipgrip
/usr/local/Cellar/pipgrip/0.5.1 (1,194 files, 11.9MB) *
  Poured from bottle on 2020-07-30 at 09:52:04
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/pipgrip.rb
License: BSD-3-Clause
==> Dependencies
Required: python@3.8 ✔
==> Analytics
install: 61 (30 days), 61 (90 days), 61 (365 days)
install-on-request: 61 (30 days), 61 (90 days), 61 (365 days)
build-error: 0 (30 days)
kbrazil-mac:~ kbrazil$ pipgrip --json jc==1.13.1 -vv
INFO: fact: _root_ is root
INFO: derived: root
INFO: fact: root depends on jc (==1.13.1)
INFO: selecting . (0.0.0)
INFO: derived: jc (==1.13.1)
INFO: fact: jc (1.13.1) depends on pygments (>=2.3.0)
INFO: fact: jc (1.13.1) depends on ruamel-yaml (>=0.15.0)
INFO: fact: jc (1.13.1) depends on xmltodict (>=0.12.0)
INFO: selecting jc (1.13.1)
INFO: derived: xmltodict (>=0.12.0)
INFO: derived: ruamel-yaml (>=0.15.0)
INFO: derived: pygments (>=2.3.0)
Traceback (most recent call last):
  File "/usr/local/bin/pipgrip", line 33, in <module>
    sys.exit(load_entry_point('pipgrip==0.5.1', 'console_scripts', 'pipgrip')())
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/cli.py", line 351, in main
    solution = solver.solve()
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/version_solver.py", line 69, in solve
    if not self._run():
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/version_solver.py", line 85, in _run
    next_package = self._choose_package_version()
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/version_solver.py", line 349, in _choose_package_version
    term = self._next_term_to_try()
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/version_solver.py", line 337, in _next_term_to_try
    term = min(*unsatisfied, key=_get_min)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/version_solver.py", line 324, in _get_min
    versions = self._source.versions_for(
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/libs/mixology/package_source.py", line 77, in versions_for
    return self._versions_for(package, constraint)
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/package_source.py", line 160, in _versions_for
    self.discover_and_add(package.req.__str__())
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/package_source.py", line 125, in discover_and_add
    to_create = discover_dependencies_and_versions(
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/pipper.py", line 364, in discover_dependencies_and_versions
    wheel_fname = _download_wheel(
  File "/usr/local/Cellar/pipgrip/0.5.1/libexec/lib/python3.8/site-packages/pipgrip/pipper.py", line 300, in _download_wheel
    raise RuntimeError("Failed to download wheel for {}".format(package))
RuntimeError: Failed to download wheel for xmltodict>=0.12.0
```

Rylan12 commented:
How about `pipgrip --json jc==1.13.1 -vv --no-cache-dir`?

kellyjonbrazil commented:
```
kbrazil-mac:~ kbrazil$ pipgrip --json jc==1.13.1 -vv --no-cache-dir
INFO: fact: _root_ is root
INFO: derived: root
INFO: fact: root depends on jc (==1.13.1)
INFO: selecting . (0.0.0)
INFO: derived: jc (==1.13.1)
INFO: fact: jc (1.13.1) depends on pygments (>=2.3.0)
INFO: fact: jc (1.13.1) depends on ruamel-yaml (>=0.15.0)
INFO: fact: jc (1.13.1) depends on xmltodict (>=0.12.0)
INFO: selecting jc (1.13.1)
INFO: derived: xmltodict (>=0.12.0)
INFO: derived: ruamel-yaml (>=0.15.0)
INFO: derived: pygments (>=2.3.0)
INFO: selecting xmltodict (0.12.0)
INFO: selecting pygments (2.6.1)
INFO: fact: ruamel-yaml (0.16.10) depends on ruamel-yaml-clib (>=0.1.2)
INFO: selecting ruamel-yaml (0.16.10)
INFO: derived: ruamel-yaml-clib (>=0.1.2)
INFO: selecting ruamel-yaml-clib (0.2.0)
INFO: Version solving took 37.664 seconds.
INFO: Tried 1 solutions.
{"jc": "1.13.1", "pygments": "2.6.1", "ruamel-yaml": "0.16.10", "ruamel-yaml-clib": "0.2.0", "xmltodict": "0.12.0"}
```

Rylan12 commented:
Thanks! It looks like `pipgrip` still needs to `--no-cache-dir` flag to be passed. We thought it wasn't needed anymore so it was removed. I'll add it back so this will work in the future.

@ddelange I'm just letting you know that we've come across another `pipgrip` `wheel` issue similar to https://github.com/ddelange/pipgrip/issues/27.  I can't recreate it on my machine, but I've verified that `pipgrip` version 0.5.1 is being used. There's no rush for a fix from Homebrew's point of view because we can add the `--no-cache-dir` flag, so don't worry about if this isn't an issue you're interested in/able to work on. I just figured I'd bring it to your attention.

kellyjonbrazil commented:
Right on - thanks!

Rylan12 commented:
Thanks, @kellyjonbrazil!

ddelange commented:
Interesting, I'm curious what changes in pip's output (I thought it was the lowercase default cache dir). @kellyjonbrazil any chance you could run the breaking command with `-vvv` on your system and share the output? Triple v will give debug statements of your env (OS, pip version etc) and the `pip wheel` output for which pipgrip failed to parse the wheel filename, which would help debug this issue! 
cc @Rylan12 for future reference :)

kellyjonbrazil commented:
No problem, here you go:

```
$ pipgrip --json jc==1.13.1 -vvv --no-cache-dir
DEBUG: Environment: {'implementation_name': 'cpython', 'implementation_version': '3.8.5', 'os_name': 'posix', 'platform_machine': 'x86_64', 'platform_release': '18.7.0', 'platform_system': 'Darwin', 'platform_version': 'Darwin Kernel Version 18.7.0: Thu Jun 18 20:50:10 PDT 2020; root:xnu-4903.278.43~1/RELEASE_X86_64', 'python_full_version': '3.8.5', 'platform_python_implementation': 'CPython', 'python_version': '3.8', 'sys_platform': 'darwin'}
DEBUG: Pip version: [20, 1, 1]
DEBUG: Downloading/building wheel for jc==1.13.1
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--wheel-dir', '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'jc==1.13.1']
DEBUG: {'jc==1.13.1': '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/jc-1.13.1-py3-none-any.whl'}
DEBUG: Searching metadata in /var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/jc-1.13.1-py3-none-any.whl
DEBUG: Finding possible versions for jc
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'jc==rubbish']
DEBUG: {'jc': ['0.2.0', '0.5.5', '0.6.4', '0.8.1', '0.9.1', '1.0.1', '1.1.1', '1.5.1', '1.6.1', '1.7.1', '1.7.2', '1.7.3', '1.7.4', '1.7.5', '1.8.0', '1.8.1', '1.9.0', '1.9.1', '1.9.2', '1.9.3', '1.10.0', '1.10.1', '1.10.2', '1.10.3', '1.10.4', '1.10.5', '1.10.6', '1.10.7', '1.10.8', '1.10.9', '1.10.10', '1.10.11', '1.10.12', '1.11.0', '1.11.1', '1.11.2', '1.11.3', '1.11.4', '1.11.5', '1.11.6', '1.11.7', '1.11.8', '1.12.0', '1.12.1', '1.13.0', '1.13.1']}
INFO: fact: _root_ is root
INFO: derived: root
INFO: fact: root depends on jc (==1.13.1)
INFO: selecting . (0.0.0)
INFO: derived: jc (==1.13.1)
INFO: fact: jc (1.13.1) depends on pygments (>=2.3.0)
INFO: fact: jc (1.13.1) depends on ruamel-yaml (>=0.15.0)
INFO: fact: jc (1.13.1) depends on xmltodict (>=0.12.0)
INFO: selecting jc (1.13.1)
INFO: derived: xmltodict (>=0.12.0)
INFO: derived: ruamel-yaml (>=0.15.0)
INFO: derived: pygments (>=2.3.0)
DEBUG: Downloading/building wheel for xmltodict>=0.12.0
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--wheel-dir', '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'xmltodict>=0.12.0']
DEBUG: {'xmltodict>=0.12.0': '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/xmltodict-0.12.0-py2.py3-none-any.whl'}
DEBUG: Searching metadata in /var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/xmltodict-0.12.0-py2.py3-none-any.whl
DEBUG: Finding possible versions for xmltodict
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'xmltodict==rubbish']
DEBUG: {'xmltodict': ['0.2', '0.3', '0.4', '0.4.1', '0.4.2', '0.4.3', '0.4.4', '0.4.6', '0.5.0', '0.5.1', '0.6.0', '0.7.0', '0.8.1', '0.8.2', '0.8.3', '0.8.4', '0.8.5', '0.8.6', '0.8.7', '0.9.0', '0.9.1', '0.9.2', '0.10.0', '0.10.1', '0.10.2', '0.11.0', '0.12.0']}
DEBUG: Downloading/building wheel for ruamel-yaml>=0.15.0
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--wheel-dir', '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'ruamel-yaml>=0.15.0']
DEBUG: {'ruamel-yaml>=0.15.0': '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/ruamel.yaml-0.16.10-py2.py3-none-any.whl'}
DEBUG: Searching metadata in /var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/ruamel.yaml-0.16.10-py2.py3-none-any.whl
DEBUG: included conditional dep ruamel.yaml.clib (>=0.1.2) ; platform_python_implementation=="CPython" and python_version<"3.9"
DEBUG: dropped conditional dep ruamel.ordereddict ; platform_python_implementation=="CPython" and python_version<="2.7"
DEBUG: dropped conditional dep ryd ; extra == 'docs'
DEBUG: dropped conditional dep ruamel.yaml.jinja2 (>=0.2) ; extra == 'jinja2'
DEBUG: Finding possible versions for ruamel-yaml
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'ruamel-yaml==rubbish']
DEBUG: {'ruamel-yaml': ['0.1', '0.2', '0.3', '0.4', '0.5', '0.5.1', '0.6', '0.6.1', '0.7', '0.7.1', '0.7.2', '0.8', '0.9.1', '0.9.2', '0.9.4', '0.9.5', '0.9.6', '0.9.7', '0.9.8', '0.10', '0.10.1', '0.10.2', '0.10.4', '0.10.5', '0.10.6', '0.10.7', '0.10.8', '0.10.9', '0.10.10', '0.10.11', '0.10.12', '0.10.13', '0.10.14', '0.10.15', '0.10.16', '0.10.17', '0.10.18', '0.10.19', '0.10.20', '0.10.21', '0.10.22', '0.10.23', '0.11.0', '0.11.1', '0.11.2', '0.11.3', '0.11.5', '0.11.6', '0.11.7', '0.11.8', '0.11.9', '0.11.10', '0.11.11', '0.11.14', '0.11.15', '0.12.0', '0.12.1', '0.12.2', '0.12.3', '0.12.4', '0.12.5', '0.12.6', '0.12.7', '0.12.8', '0.12.9', '0.12.10', '0.12.11', '0.12.12', '0.12.13', '0.12.14', '0.12.15', '0.12.16', '0.12.17', '0.12.18', '0.13.0', '0.13.1', '0.13.2', '0.13.3', '0.13.4', '0.13.5', '0.13.6', '0.13.7', '0.13.8', '0.13.9', '0.13.10', '0.13.11', '0.13.13', '0.13.14', '0.14.0', '0.14.1', '0.14.2', '0.14.3', '0.14.4', '0.14.5', '0.14.6', '0.14.7', '0.14.8', '0.14.9', '0.14.10', '0.14.11', '0.14.12', '0.15.0', '0.15.1', '0.15.2', '0.15.3', '0.15.4', '0.15.5', '0.15.6', '0.15.7', '0.15.8', '0.15.9', '0.15.10', '0.15.11', '0.15.12', '0.15.13', '0.15.14', '0.15.15', '0.15.16', '0.15.17', '0.15.18', '0.15.19', '0.15.20', '0.15.21', '0.15.22', '0.15.23', '0.15.24', '0.15.25', '0.15.26', '0.15.27', '0.15.28', '0.15.29', '0.15.30', '0.15.31', '0.15.32', '0.15.33', '0.15.34', '0.15.35', '0.15.37', '0.15.38', '0.15.39', '0.15.40', '0.15.41', '0.15.42', '0.15.43', '0.15.44', '0.15.45', '0.15.46', '0.15.47', '0.15.48', '0.15.49', '0.15.50', '0.15.51', '0.15.52', '0.15.53', '0.15.54', '0.15.55', '0.15.56', '0.15.57', '0.15.58', '0.15.59', '0.15.60', '0.15.61', '0.15.62', '0.15.63', '0.15.64', '0.15.65', '0.15.66', '0.15.67', '0.15.68', '0.15.69', '0.15.70', '0.15.71', '0.15.72', '0.15.73', '0.15.74', '0.15.75', '0.15.76', '0.15.77', '0.15.78', '0.15.79', '0.15.80', '0.15.81', '0.15.82', '0.15.83', '0.15.84', '0.15.85', '0.15.86', '0.15.87', '0.15.88', '0.15.89', '0.15.90', '0.15.91', '0.15.92', '0.15.93', '0.15.94', '0.15.95', '0.15.96', '0.15.97', '0.15.98', '0.15.99', '0.15.100', '0.16.0', '0.16.1', '0.16.2', '0.16.3', '0.16.4', '0.16.5', '0.16.6', '0.16.7', '0.16.9', '0.16.10']}
DEBUG: Downloading/building wheel for pygments>=2.3.0
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--wheel-dir', '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'pygments>=2.3.0']
DEBUG: {'pygments>=2.3.0': '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/Pygments-2.6.1-py3-none-any.whl'}
DEBUG: Searching metadata in /var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/Pygments-2.6.1-py3-none-any.whl
DEBUG: Finding possible versions for pygments
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'pygments==rubbish']
DEBUG: {'pygments': ['0.5', '0.5.1', '0.6', '0.7', '0.7.1', '0.8', '0.8.1', '0.9', '0.10', '0.11', '0.11.1', '1.0', '1.1', '1.1.1', '1.2', '1.2.1', '1.2.2', '1.3', '1.3.1', '1.4', '1.5', '1.6', '2.0', '2.0.1', '2.0.2', '2.1', '2.1.1', '2.1.2', '2.1.3', '2.2.0', '2.3.0', '2.3.1', '2.4.0', '2.4.1', '2.4.2', '2.5.1', '2.5.2', '2.6.0', '2.6.1']}
INFO: selecting xmltodict (0.12.0)
INFO: selecting pygments (2.6.1)
INFO: fact: ruamel-yaml (0.16.10) depends on ruamel-yaml-clib (>=0.1.2)
INFO: selecting ruamel-yaml (0.16.10)
INFO: derived: ruamel-yaml-clib (>=0.1.2)
DEBUG: Downloading/building wheel for ruamel-yaml-clib>=0.1.2
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--wheel-dir', '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'ruamel-yaml-clib>=0.1.2']
DEBUG: {'ruamel-yaml-clib>=0.1.2': '/var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/ruamel.yaml.clib-0.2.0-cp38-cp38-macosx_10_14_x86_64.whl'}
DEBUG: Searching metadata in /var/folders/vr/4gybj0rs1_51r0sy8d5qv3jm0000gn/T/tmpwq6koqle/ruamel.yaml.clib-0.2.0-cp38-cp38-macosx_10_14_x86_64.whl
DEBUG: Finding possible versions for ruamel-yaml-clib
DEBUG: ['/usr/local/Cellar/pipgrip/0.5.1/libexec/bin/python3.8', '-m', 'pip', 'wheel', '--no-deps', '--disable-pip-version-check', '--index-url', 'https://pypi.org/simple', '--trusted-host', 'pypi.org', '--progress-bar=off', 'ruamel-yaml-clib==rubbish']
DEBUG: {'ruamel-yaml-clib': ['0.1.0', '0.1.2', '0.2.0']}
INFO: selecting ruamel-yaml-clib (0.2.0)
INFO: Version solving took 11.739 seconds.
INFO: Tried 1 solutions.
DEBUG: OrderedDict([(Package("jc"), <Version 1.13.1>), (Package("xmltodict"), <Version 0.12.0>), (Package("pygments"), <Version 2.6.1>), (Package("ruamel-yaml"), <Version 0.16.10>), (Package("ruamel-yaml-clib"), <Version 0.2.0>)])
{"jc": "1.13.1", "pygments": "2.6.1", "ruamel-yaml": "0.16.10", "ruamel-yaml-clib": "0.2.0", "xmltodict": "0.12.0"}
```
